src/main/java/io/excaliburfrc/robot/subsystems/Climber.java

```diff
@@ -0,0 +1,88 @@
+package io.excaliburfrc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj2.command.*;
+import io.excaliburfrc.robot.Constants;
+
+public class Climber extends SubsystemBase implements AutoCloseable {
+  private final DoubleSolenoid anglerPiston =
+      new DoubleSolenoid(
+          PneumaticsModuleType.CTREPCM,
+          Constants.ClimberConstants.FORWARD_CHANNEL,
+          Constants.ClimberConstants.REVERSE_CHANNEL);
+  private final CANSparkMax motor =
+      new CANSparkMax(
+          Constants.ClimberConstants.CLIMBER_SPARKMAX, CANSparkMaxLowLevel.MotorType.kBrushless);
+  private final DigitalInput sensor = new DigitalInput(Constants.ClimberConstants.SENSOR_CHANNEL);
```

Comment:
Please import `ClimberConstants` itself, or star-static-import the constants directly.